tests/unit/test_project.py

```diff
@@ -406,9 +406,10 @@ def test_full_service():
         "user-id": 1000,
         "group": "ubuntu",
         "group-id": 1000,
+        "working-dir": "/tmp",
         "on-success": "ignore",
         "on-failure": "restart",
-        "on-check_failure": {"check": "restart"},
+        "on-check-failure": {"check": "restart"},
```

Comment:
How can the test pass before with the -? it's was because the ``extra=forbid`` was not here?

Reply:
yep